### PR TITLE
search logo

### PR DESCRIPTION
**Pull request by jdorweiler**
**Reviewer:**

## Description:

Fix missing search box logo in Firefox.
screen shot 2017-06-16 at 11 32 15 am

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: css/_vars.scss [hunk 1]
```diff
@@ -8,6 +8,7 @@ $color--neutral-bg: #f5f5f5;
 
 /* Icons & images */
 $ddg-logo__url: "chrome-extension://__MSG_@@extension_id__/img/logo-horizontal@2x.png";
```
Comment (laurengarcia) [inline css/_vars.scss]: How about `$ddg-logo__url--chrome` (double dashes for variations on a thing).

File: css/_vars.scss [hunk 2]
```diff
@@ -8,6 +8,7 @@ $color--neutral-bg: #f5f5f5;
...
+$ddg-logo__url-moz: "moz-extension://__MSG_@@extension_id__/img/logo-horizontal@2x.png";
```
Comment (laurengarcia) [inline css/_vars.scss]: ... and `$ddg-logo__url--moz` here...

File: css/trackers.scss [hunk 3]
```diff
@@ -98,6 +98,16 @@
...
+        .ddg-logo-moz {
+            background-image: url($ddg-logo__url-moz);
+            background-size: cover;
+            width: 25px;
+            height: 25px;
+            position: absolute;
+            top: 7px;
+            left: 7px;
+        }
+
```
Comment (laurengarcia) [inline css/trackers.scss]: ...then here, just leave all the rules except the `background-image` in a generic `.ddg-logo` rule. Then within `.ddg-logo { .... }` rules, nest each class, both `.ddg-logo--moz` and `.ddg-logo--chrome` (<-- note two dashes) with the background-image url for each.

File: js/templates/search.es6.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ module.exports = function () {
           <input type="text" autocomplete="off" placeholder="Search DuckDuckGo" name="q" class="search-form__input js-search-input" value="${this.model.searchText}" />
           <input class="search-form__go js-search-go" tabindex="2" value="" type="button" />
           <input type="submit" class="search-form__submit" />
-          <span class="ddg-logo"></span>
+          <span class="ddg-logo-${this.model.browser}"></span>
```
Comment (laurengarcia) [inline js/templates/search.es6.js]: ... now you can do `<span class="ddg-logo ddg-logo--${this.model.browser}"... ` here.

Comment (russellholt) [inline js/templates/search.es6.js]: 👍

**Comment by russellholt** [comment]
This will be addressed a different way.

**Pull request closed without merging**